config/install/field.storage.user.field_age_group.yml

```diff
@@ -11,7 +11,7 @@ type: list_integer
 settings:
   allowed_values:
     -
-      value: 1939
+      value: 1929
```

Comment:
Why do we change it?

Reply:
There is was a problem with d3 scale tickets and to show the 1930 year needed to change it.